### DISCUSSION TITLE
怎么把Github当绳网了？

### DISCUSSION THREAD

**Discussion by whatcanisaylaoda** [share121/inter-knot#1934] [其他]
![Screenshot_20240813-205616](https://github.com/user-attachments/assets/e569826d-ddd7-4270-84b8-6d75a9b10897)
这怎么回事？

**Comment by AnbulesPeo** [comment]
好像这个绳网就是github讨论区的套壳desu

**Comment by lllliufeng** [comment]
打开了github，绳网打不开了（笑

**Reply by ikungei** [reply]
同问什么情况这是

**Comment by whatcanisaylaoda** [comment]
![Uploading Screenshot_20240814-070933.png…]()
我没在绳网发。

**Comment by whatcanisaylaoda** [comment]
我还进不去呢。

**Comment by HifumiMizuhara** [comment]
作者给GitHub Discussion做了绳网的UI。
访问本仓库的GitHub Pages 即可。

**Comment by whatcanisaylaoda** [comment]
我一直在Github上。

**Reply by HifumiMizuhara** [reply]
意思就是，作者想要还原绳网，所以就用GitHub Discussion的API做了绳网。
大概就和那些用Github issues 写博客的人差不多。

**Comment by nd304** [comment]
![img2071310697_1723307286285](https://github.com/user-attachments/assets/a3bc2e46-fb6a-4787-8802-48bc6892d0da)
